code/index.html

```diff
@@ -1,30 +1,48 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Project 3</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+<body>
+  <header>
+    <div>
+      <h1>Skip Plastic, Be Fantastic!</h1>
+    </div>
+  </header>
+  <main>
+    <flex-child>
+      <form action="http://httpbin.org/anything" method="POST">
+        <h2>Create your account</h2>
+        <input type="text" name="email" placeholder="Email" required />
```

Comment:
This will automatically valide that the input value is an email!  
```suggestion
        <input type="email" name="email" placeholder="Email" required />
```